.github/workflows/license-scan.yaml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   fossa-scan:
     name: Find license compliance and security issues
-    if: github.repository == 'Kuadrant/limitador'
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Hope this fixes it 🤞 
I had this job fail here once on `main`, but let's try this 🙌